packages/dev/core/src/Gizmos/scaleGizmo.ts

```diff
@@ -155,11 +167,8 @@ export class ScaleGizmo extends Gizmo implements IScaleGizmo {
      * True when the mouse pointer is hovering a gizmo mesh
      */
     public get isHovered() {
-        let hovered = false;
-        [this.xGizmo, this.yGizmo, this.zGizmo].forEach((gizmo) => {
-            hovered = hovered || gizmo.isHovered;
-        });
-        return hovered;
+        const { xGizmo, yGizmo, zGizmo } = this;
+        return xGizmo.isHovered || yGizmo.isHovered || zGizmo.isHovered;

```

Comment:
```suggestion
        return this.xGizmo.isHovered || this.yGizmo.isHovered || this.zGizmo.isHovered;
```
I don't see the point of extracting first the values in some variables (?)

Reply:
That's how it started out
```ts
    public get isHovered() {
        let hovered = false;
        [this.xGizmo, this.yGizmo, this.zGizmo].forEach((gizmo) => {
            hovered = hovered || gizmo.isHovered;
        });
        return hovered;
    }
```
I changed it to the style below, and the new code is more readable and performant
```ts
    public get isHovered() {
        const {xGizmo,yGizmo,zGizmo} = this;
        return xGizmo.isHovered || yGizmo.isHovered || zGizmo.isHovered;
    }
```

Reply:
I understand the motivation, but why not doing the final step `return this.xGizmo.isHovered || this.yGizmo.isHovered || this.zGizmo.isHovered;`? The `const ...` line is an unncessary step, and one can argue it makes the code less readable (though, everything is a matter of taste at some point...).